.gitignore

```diff
@@ -5,4 +5,5 @@
 !.vscode/extensions.json
 *.code-workspace
 .env
-JSONKeys.txt
\ No newline at end of file
+JSONKeys.txt
+CS125.env
```

Comment:
btw, if you rename `CS125.env` to just `.env`, github will autoignore it (it might be worth the effort as long as you don't have any other `.env` files in the directory

Reply:
in that case, i don't think you would need to include the `CS125.env` in the .gitignore